composer.json

```diff
@@ -26,7 +26,7 @@
     "php": ">=5.3.0",
     "ext-curl": "*",
     "ext-json": "*",
-    "lib-curl": ">=7.20.0"
+    "lib-curl": ">=7.19.0"
```

Comment:
I think it should not let the `lib-curl` be defined version.
How about setting the `lib-curl: "*"` for this version definition?